app/scripts/modules/ecs/src/serverGroup/details/events/events.html

```diff
@@ -0,0 +1,40 @@
+<div modal-page>
+  <modal-close dismiss="$ctrl.close()"></modal-close>
+  <div class="modal-header">
+    <h3>Server Group Events for {{$ctrl.serverGroup.name}}</h3>
+  </div>
+  <div class="modal-body">
+    <div class="loading" ng-if="$ctrl.viewState.loading">
+      <loading-spinner size="'small'"></loading-spinner>
+    </div>
+    <div ng-if="$ctrl.viewState.error" class="text-center">
+      <p>There was an error loading events for {{$ctrl.serverGroup.name}}. Please try again later.</p>
+    </div>
+    <div ng-if="!$ctrl.viewState.loading && !$ctrl.viewState.error && !$ctrl.events.length" class="text-center">
+      No scaling events found for {{$ctrl.serverGroup.name}}.</p>
+    </div>
+
+    <div ng-repeat="event in $ctrl.events" ng-if="!$ctrl.viewState.loading">
+        <p class="clearfix">
+          <div ng-if="event.status == 'Success'">
```

Comment:
I would s/==/=== here and below, though it doesn't matter.

Reply:
Sorry, not very adept at Javascript.  I am not sure what you mean here :(

Reply:
instead of `ng-if="event.status == 'Success'"`, do `ng-if="event.status === 'Success'"`